src/constructs/cloudwatch/lambda-alarms.ts

```diff
@@ -46,3 +46,49 @@ export class GuLambdaErrorPercentageAlarm extends GuAlarm {
     super(scope, id, alarmProps);
   }
 }
+
+export interface GuLambdaThrottlingMonitoringProps
+  extends Omit<
+    GuAlarmProps,
+    "metric" | "threshold" | "comparisonOperator" | "evaluationPeriods" | "treatMissingData" | "app"
+  > {
+  /**
+   * Sum of thottled invocations at which to alarm.
+   *
+   * @defaultValue 1
+   */
+  toleratedThrottlingCount?: number;
+
+  /**
+   * Evaluation period in minutes for alarm.
+   *
+   * @defaultValue 1
+   */
+  numberOfMinutesAboveThresholdBeforeAlarm?: number;
+
+  noMonitoring?: false;
+}
+
+interface GuLambdaThrottlingAlarmProps extends GuLambdaThrottlingMonitoringProps {
+  lambda: GuLambdaFunction;
+}
+
+export class GuLambdaThrottlingAlarm extends GuAlarm {
+  constructor(scope: GuStack, id: string, props: GuLambdaThrottlingAlarmProps) {
+    super(scope, id, {
+      ...props,
+      app: props.lambda.app,
+      alarmName: props.alarmName ?? "Lambda throttling alarm",
+      alarmDescription:
+        props.alarmDescription ?? "Alarm when lambda is throttled (which causes requests to fail with a 429).",
+      threshold: props.toleratedThrottlingCount ?? 1,
+      evaluationPeriods: props.numberOfMinutesAboveThresholdBeforeAlarm ?? 1,
+      metric: props.lambda.metricThrottles({
+        period: Duration.seconds(60),
+        statistic: "sum",
+      }),
+      treatMissingData: TreatMissingData.NOT_BREACHING,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
```

Comment:
Minor: for consistency with other alarms (and because of the name `toleratedThrottlingCount`) I think we want this to be `GREATER_THAN` https://github.com/guardian/cdk/blob/b1913a5a23cf6d9c3ff64584b63feaa434fc62d3/src/constructs/cloudwatch/lambda-alarms.ts#L41

Reply:
good shout. 👍